freqtrade/optimize/optimize_reports.py

```diff
@@ -164,6 +164,12 @@ def generate_strategy_comparison(all_results: Dict) -> List[Dict]:
         tabular_data.append(_generate_result_line(
             results['results'], results['config']['dry_run_wallet'], strategy)
             )
+        try:
+            max_drawdown, _, _, _, _ = calculate_max_drawdown(results['results'],
+                                                              value_col='profit_ratio')
```

Comment:
i'm not convinced that using "profit_ratio" will give you what you want (comparable drawdowns).

As the list does not contain the average stake size - the relative result is quite ... dependent.

Consider the below ... 

```
========================================================================== STRATEGY SUMMARY =========================================================================                                                                         
|          Strategy |   Buys |   Avg Profit % |   Cum Profit % |   Tot Profit USDT |   Tot Profit % |     Avg Duration |   Wins |   Draws |   Losses |   Drawdown % |                                                                         
|-------------------+--------+----------------+----------------+-------------------+----------------+------------------+--------+---------+----------+--------------|                                                                         
| xxx |   2052 |           0.70 |        1439.62 |          9308.003 |         930.80 |          5:33:00 |   1041 |     459 |      552 |       341.61 |                                                                         
|  yyy |    169 |           4.52 |         764.42 |          1243.602 |         124.36 | 3 days, 23:04:00 |     32 |       0 |      137 |       206.2  |                                                                         
=====================================================================================================================================================================  
```


OK, drawdown is "rather" high (that's a random test) - as it's 341% and 206%.

However, looking at absolute drawdowns, it's actually 2240$ (341%)- vs. 365$ (206%) - simply because the total gains where a lot higher - so the average stake was also higher.



```
=============== SUMMARY METRICS ===============                                                                                                                                                                                               
| Metric                | Value               |                                                                                                                                                                                               
|-----------------------+---------------------|                                                                                                                                                                                               
| Backtesting from      | 2020-08-01 00:00:00 |                                                                                                                                                                                               
| Backtesting to        | 2020-12-31 23:00:00 |                                                                                                                                                                                               
| Max open trades       | 5                   |                                                                                                                                                                                               
|                       |                     |                                                                                                                                                                                               
| Total trades          | 2052                |                                                                                                                                                                                               
| Starting balance      | 1000.000 USDT       |                                                                                                                                                                                               
| Final balance         | 10308.003 USDT      |                                                                                                                                                                                               
| Absolute profit       | 9308.003 USDT       |                                                                                                                                                                                               
| Total profit %        | 930.8%              |                                                                                                                                                                                               
| Trades per day        | 13.5                |                                                                                                                                                                                               
| Avg. stake amount     | 789.349 USDT        |                                                                                                                                                                                               
| Total trade volume    | 1619743.675 USDT    |                                                                                                                                                                                               
|                       |                     |                                                                                                                                                                                               
| Best Pair             | XRPUP/USDT 295.63%  |                                                                                                                                                                                               
| Worst Pair            | EGLD/USDT -50.35%   |                                                                                                                                                                                               
| Best trade            | XRPUP/USDT 241.49%  |                                                                                                                                                                                               
| Worst trade           | LINKUP/USDT -10.18% |                                                                                                                                                                                               
| Best day              | 1616.762 USDT       |                                                                                                                                                                                               
| Worst day             | -954.338 USDT       |                                                                                                                                                                                               
| Days win/draw/lose    | 89 / 1 / 63         |                                                                                                                                                                                               
| Avg. Duration Winners | 4:14:00             |                                                                                                                                                                                               
| Avg. Duration Loser   | 6:07:00             |                                                                                                                                                                                               
| Zero Duration Trades  | 19.4% (398)         |                                                                                                                                                                                               
|                       |                     |                                                                                                                                                                                               
| Min balance           | 988.668 USDT        |                                                                                                                                                                                               
| Max balance           | 11083.536 USDT      |                                                                                                                                                                                               
| Drawdown              | 341.61%             |                                                                                                                                                                                               
| Drawdown              | 2247.062 USDT       |                                                                                                                                                                                               
| Drawdown high         | 3121.184 USDT       |                                                                                                                                                                                               
| Drawdown low          | 874.122 USDT        |                                                                                                                                                                                               
| Drawdown Start        | 2020-08-31 17:00:00 |                                                                                                                                                                                               
| Drawdown End          | 2020-11-07 12:00:00 |                                                                                                                                                                                               
| Market change         | 31.37%              |                                                                                                                                                                                               
=============================================== 

.....

================ SUMMARY METRICS ================                                                                                                                                                                                             
| Metric                | Value                 |                                                                                                                                                                                             
|-----------------------+-----------------------|                                                                                                                                                                                             
| Backtesting from      | 2020-08-01 00:00:00   |                                                                                                                                                                                             
| Backtesting to        | 2020-12-31 23:00:00   |                                                                                                                                                                                             
| Max open trades       | 5                     |                                                                                                                                                                                             
|                       |                       |                                                                                                                                                                                             
| Total trades          | 169                   |                                                                                                                                                                                             
| Starting balance      | 1000.000 USDT         |                                                                                                                                                                                             
| Final balance         | 2243.602 USDT         |                                                                                                                                                                                             
| Absolute profit       | 1243.602 USDT         |                                                                                                                                                                                             
| Total profit %        | 124.36%               |                                                                                                                                                                                             
| Trades per day        | 1.11                  |                                                                                                                                                                                             
| Avg. stake amount     | 196.826 USDT          |                                                                                                                                                                                             
| Total trade volume    | 33263.552 USDT        |                                                                                                                                                                                             
|                       |                       |                                                                                                                                                                                             
| Best Pair             | BTCUP/USDT 256.15%    |                                                                                                                                                                                             
| Worst Pair            | LINKDOWN/USDT -72.66% |                                                                                                                                                                                             
| Best trade            | CVC/USDT 244.32%      |                                                                                                                                                                                             
| Worst trade           | SXP/USDT -5.19%       |                                                                                                                                                                                             
| Best day              | 1049.895 USDT         |                                                                                                                                                                                             
| Worst day             | -67.928 USDT          |                                                                                                                                                                                             
| Days win/draw/lose    | 20 / 83 / 50          |                                                                                                                                                                                             
| Avg. Duration Winners | 14 days, 13:52:00     |                                                                                                                                                                                             
| Avg. Duration Loser   | 1 day, 11:33:00       |                                                                                                                                                                                             
| Zero Duration Trades  | 11.8% (20)            |                                                                                                                                                                                             
|                       |                       |                                                                                                                                                                                             
| Min balance           | 721.943 USDT          |                                                                                                                                                                                             
| Max balance           | 2243.602 USDT         |                                                                                                                                                                                             
| Drawdown              | 206.2%                |                                                                                                                                                                                             
| Drawdown              | 363.004 USDT          |                                                                                                                                                                                             
| Drawdown high         | 84.947 USDT           |                                                                                                                                                                                             
| Drawdown low          | -278.057 USDT         |                                                                                                                                                                                             
| Drawdown Start        | 2020-08-21 03:00:00   |                                                                                                                                                                                             
| Drawdown End          | 2020-10-08 14:00:00   |                                                                                                                                                                                             
| Market change         | 31.37%                |                                                                                                                                                                                             
=================================================                 

```

to give this value any meaning (in case of unrelated strategies, anyway - it gets better if results are rather similar), we'll probably have to have avg stake amount as well.

Reply:
I suspect people will not understand why it does not match drawdown in summary metrics

Reply:
if you use absolute drawdown it would ...

what would be the alternative you're thinking about now?

Reply:
maybe absolute + profit ratio then? idk i keep looking at profit ratio drawdown when comparing different versions of same strategy. if it was only absolute value it would be bit harder to see how much drawdown changed/ % is convenient for similar strategies. absolute values... hmm i dont see them convenient to me at all, but it may be just me

Reply:
as said above - it'll depend - if you have very similar strategies - % values will be fine.
if strategies are utterly different (as in my case above) - then % change will not give you too much.

Reply:
Added absolute drawdown value as well. Table is getting a bit long though.

![image](https://user-images.githubusercontent.com/19151258/118389031-799dc400-b630-11eb-8b5c-d5c08579382d.png)

Reply:
Maybe we can/should combine Wins / Draws / Losses into one column (like it is for hyperopt)

there's little point in having 3 columns ... obviously this would apply to the "per pair" table too - but it'll even become easier to read as on my (half-screen width) terminal, the lines wrap anyway ...

Reply:
sounds good to me. maybe we should do same for drawdown as well? Those two columns waste a lot of space